src/main/java/frc/robot/Constants.java

```diff
@@ -82,21 +98,22 @@ public static final class Swerve {
 
         /* Swerve Profiling Values */
         /** Meters per Second */
-        public static final double maxSpeed = 4.5; //TODO: This must be tuned to specific robot
+        //4.5
+        public static final double maxSpeed = 4.5*0.8; //TODO: This must be tuned to specific robot
```

Comment:
Why did we add this? Also, investigate where this is used. Is this why we seemed a little slow in Sussex?